client/tutorials.json

```diff
@@ -0,0 +1,47 @@
+{
+    "series":[
+        {
+            "name": "Introduction",
+            "length": "3 hours",
+            "difficulty": "simple",
+            "description": "Series of tutorials that will introduce the basic concepts",
+            "tutorials": [
+                {
+                    "title": "Local smart contract development",
+                    "length": "30 mins",
+                    "source": "https://developer.ibm.com/tutorials/ibm-blockchain-platform-vscode-smart-contract",
+                    "description": "Follow the typical workflow from generating a new smart contract project, deploying code to the local_fabric runtime, and testing your transactions via an application gateway.",
+                    "file": "ibm-blockchain-platform-vscode-smart-contract/index.md"
```

Comment:
can we make the index.md be tutorial.md so that when it opens it shows preview tutorial.md rather than index.md

Reply:
How about the actual tutorial name? E.g `Local smart contract development.md`. This might be useful for keeping track if a user has multiple tutorials tabs open.

Spaces in the filename seem to be okay on Mac, but not sure if naming it like this will cause problems on Linux and Windows.

Reply:
sure we can have the tutorial name